src/usr/local/www/vpn_ipsec.php

```diff
@@ -289,25 +308,6 @@
 						<td>
 <?php
 			if ($ph1ent['interface']) {
```

Comment:
While you are changing it please also check if `$iflabels[$ph1ent['interface']]` is defined